<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>triwindow</class>
 <widget class="QWidget" name="triwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>管理员</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>操作</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="deletePushButton">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="recoverPushButton">
            <property name="text">
             <string>恢复</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="searchPushButton">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="savePushButton">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="statisticPushButton">
            <property name="text">
             <string>统计</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="backPushButton">
            <property name="text">
             <string>返回</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <widget class="QWidget" name="addPage">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>添加</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>身份证号</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QComboBox" name="startProvince">
                <item>
                 <property name="text">
                  <string>北京</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>河北</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>姓名</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="4">
               <widget class="QLineEdit" name="passengerName"/>
              </item>
              <item row="4" column="3" colspan="2">
               <widget class="QComboBox" name="seatComboBox">
                <item>
                 <property name="text">
                  <string>硬座</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>卧铺</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>站票</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>高铁</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>终点站</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>席别</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>始发站</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" rowspan="2" colspan="4">
               <widget class="QLineEdit" name="passengerId"/>
              </item>
              <item row="6" column="0">
               <widget class="QComboBox" name="endProvince">
                <item>
                 <property name="text">
                  <string>北京</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>河北</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="6" column="3" colspan="2">
               <widget class="QSpinBox" name="numberSpinBox"/>
              </item>
              <item row="7" column="0" colspan="5">
               <widget class="QCalendarWidget" name="calendarWidget"/>
              </item>
              <item row="6" column="1" colspan="2">
               <widget class="QComboBox" name="endCity"/>
              </item>
              <item row="8" column="0" colspan="5">
               <widget class="QPushButton" name="addPushButton_2">
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>购票张数</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="2">
               <widget class="QComboBox" name="startCity"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="deletePage">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>删除</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>输入序号</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="deletLineEdit">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="deletePushButton_2">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="recoverPage"/>
         <widget class="QWidget" name="searchPage">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>查询</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>查找方式</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="mannerComboBox">
                <item>
                 <property name="text">
                  <string>身份证号</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>姓名</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="mannerLabel">
                <property name="text">
                 <string>身份证号</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="searchLineEdit">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QPushButton" name="searchPushButton_2">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QPushButton" name="showAll">
                <property name="text">
                 <string>显示全部</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="modifyPage">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>请直接在右侧列表中进行修改，然后点击保存</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="statisticPage">
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>统计</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0" colspan="4">
               <widget class="QCalendarWidget" name="calendarWidget_3"/>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>当日订票张数</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="2" colspan="2">
               <widget class="QLabel" name="showAmountLabel">
                <property name="text">
                 <string>0张</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>当日订票总金额</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="2" colspan="2">
               <widget class="QLabel" name="showMoneyLabel">
                <property name="text">
                 <string>0￥</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>统计单价</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="showMoenyLabel_2">
                <property name="text">
                 <string>0￥</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="4">
               <widget class="QPushButton" name="statisticPushButton_2">
                <property name="text">
                 <string>统计</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="priceComboBox">
                <item>
                 <property name="text">
                  <string>小于100元</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>大于等于100元且小于等于400元</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>大于400元</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="showAmountLabel_2">
                <property name="text">
                 <string>0张</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
